### ISSUE TITLE
Solr search and variable level metadata

### ISSUE THREAD

**Issue by lubitchv** [IQSS/dataverse#6545]
With Data Curation Tool it is possible to update variable level metadata, but presently, updates in variable label are not reflected in solr search. 
There are also new variable level fields such as literal question, interview instructions, post question, universe and notes that would be needed to add to solr schema and that solr search would be able to reflect updates in these fields.

**Comment by pdurbin** [comment]
Right. Some fields in are searchable using Advanced Search:

<img width="745" alt="Screen Shot 2020-01-21 at 3 33 01 PM" src="https://user-images.githubusercontent.com/21006/72841015-7c1e2280-3c63-11ea-9f4a-bfc571136175.png">

And some other fields are indexed, such as "universe", but not in Advanced Search.

    <field name="universe" type="text_en" multiValued="true" stored="true" indexed="true"/>

Are you thinking about making a pull request? 😄

**Comment by lubitchv** [comment]
Yes, I am planning to make a pull request soon. Since updated variable label is now located in variablemetadata table and plus additional new fields there, they are not reflected in solr search. Also I think indexed "universe" is for dataset level and now there is a variable level "universe".

**Comment by pdurbin** [comment]
@lubitchv oh, you're right. "universe" is coming from the dataset level. Thanks. If you need anything, please let us know!

**Comment by djbrooke** [comment]
@lubitchv, we unfortunately had to revert the changes related to indexing as it slowed indexing down to the point where it would no longer complete. This reversion was done in #6704 and @sekmiller should be able to provide more information if needed. We'd still like to get this to Dataverse, but it was a blocker for the 4.20 release, so we pulled it out for now. I just added this comment to #6577 as well.
